src/rest/routes/github-callback/service.ts

```diff
@@ -0,0 +1,34 @@
+import { Request } from "express";
+import { envVars } from "config/env";
+import axios from "axios";
+
+const GITHUB_CLOUD_LOGIN_URL = "https://github.com/login/oauth/access_token";
+export const getAccessToken = async (req: Request) => {
```

Comment:
Put a proper typing on the return value please.

Reply:
Updated